### DISCUSSION TITLE
camelCasing becomes camel-casing

### DISCUSSION THREAD

**Discussion by oDinZu** [ComfortablyCoding/strapi-plugin-slugify#74] [Q&A] [answered]
I am unsure if this is a bug or a feature or both, dependent upon ones relative spacetime needs. 

The slugify option turns camelCasing into camel-casing lowercase with the hyphen for friendly URL's. 

It took the title of a webpage called WebPage and split the title of the post into web-page and **created unfriendly post search engine optimization.**

**Comment by ComfortablyCoding** [comment, marked as answer]
This is due to the default options of the underlying library being used.

If you add the [lowercase](https://github.com/sindresorhus/slugify#lowercase) and [decamilize](https://github.com/sindresorhus/slugify#decamelize) options as false in the plugins `slugifyOptions` property it should produce the slug as you want it.
